tests/settings/ConfigurationGatheringTest.py

```diff
@@ -369,13 +373,20 @@ def test_get_filtered_bears(self):
 
         self.assertEqual(len(local_bears['cli']), TEST_BEARS_COUNT)
 
+        self.assertEqual(
+            [str(bear) for bear in local_bears['cli']],
+            TEST_BEAR_NAMES)
+
         with bear_test_module():
             local_bears, global_bears = get_filtered_bears(
                 ['Java'], self.log_printer)
 
-        self.assertEqual(len(local_bears['cli']), 2)
-        self.assertEqual(str(local_bears['cli'][0]),
-                         "<class 'JavaTestBear.JavaTestBear'>")
-        self.assertEqual(str(local_bears['cli'][1]),
-                         "<class 'LineCountTestBear.LineCountTestBear'>")
+        expected = [
+            "<class 'JavaTestBear.JavaTestBear'>",
+            "<class 'LineCountTestBear.LineCountTestBear'>",
+        ]
+
+        self.assertEqual(
+            [str(bear) for bear in local_bears['cli']],
+            expected)
```

Comment:
I didnt notice it was a rewrite; there were a few iterations before the order became the same as the previous test order.

I think I'll just revert, as this doesnt add much.